lib/active_record/connection_adapters/postgis/postgis_database_tasks.rb

```diff
@@ -96,11 +96,11 @@ def extension_names
         end
 
         def ensure_installation_configs
-          if configuration["setup"] == "default" && !configuration["postgis_extension"]
+          if configuration_hash[:setup] == "default" && !configuration_hash[:postgis_extension]
             share_dir = `pg_config --sharedir`.strip rescue "/usr/share"
             control_file = ::File.expand_path("extension/postgis.control", share_dir)
             if ::File.readable?(control_file)
-              configuration["postgis_extension"] = "postgis"
+              configuration_hash[:postgis_extension] = "postgis"
```

Comment:
This is not possible anymore, because `configuration_hash` is frozen

Ref: https://travis-ci.org/github/rgeo/activerecord-postgis-adapter/builds/749304801